src/main/java/org/mitre/stix/DocumentUtilities.java

```diff
@@ -35,21 +35,31 @@
 
 /**
  * A collection of utility helper methods.
- * 
+ *
  * @author nemonik (Michael Joseph Walsh <github.com@nemonik.com>)
  */
 public class DocumentUtilities {
 
 	private static final String XML_SCHEMA_INSTANCE = "http://www.w3.org/2001/XMLSchema-instance";
 	private static final String XML_NAMESPACE = "http://www.w3.org/2000/xmlns/";
 
+	private static final JAXBContext STIX_CONTEXT = initDefaultContext();
+
+	private static JAXBContext initDefaultContext(){
+		try {
+			return JAXBContext.newInstance("org.mitre.stix.stix_1");
```

Comment:
I do not appear to be as familiar with the JAXB contextPath's class scanning and loading process as you.  I do agree with your point on the "org.mitre" and working with the simple vanilla STIX package and cybox indicator element.

I'm not a maintainer and/or familiar with the STIX v1 binding jar as a whole.  Inspecting the built binding jar and the generated classes, there are definitely other package namespaces that do not start with "org.mitre" and their classes use/call the DocumentUtilitites.toDocument() utility method.  Theoretically, they could possibly fall into the same issue of always instantiating a new JAXBContext.

This comes back to the possible recommendation of DocumentUtilities just maintaining an internal lazy loaded cache of called upon JAXBContext objects for marshaling.   This approach would make that one line of code not so coupled/fixed to the context namespace of "org.mitre.stix.stix_1" and starts with "org.mitre" condition. This is probably minor, but just a suggestion to cover other unexpected/possible usage of this binding jar.